app/src/main/java/eu/kanade/tachiyomi/data/database/models/SourceCount.kt

```diff
@@ -0,0 +1,3 @@
+package eu.kanade.tachiyomi.data.database.models
+
+data class SourceCount(val source: Long, val count: Int)
```

Comment:
Could be more descriptive
```suggestion
data class SourceIdWithCount(val sourceId: Long, val count: Int)
```